scripts/install_and_test_openqaoa_azure.sh

```diff
@@ -16,8 +16,4 @@ for entry in "${modulesList[@]}"; do
     cd src/$entry
     pytest -n auto tests
     cd "../.."
-done
```

Comment:
Can you help me understand why we are not uninstalling packages anymore after testing?

Reply:
I think in the end, the only use-case for the scripts are for situations where the user wants to install the modules with the relevant unittests and run the tests. 

The uninstallation became double work for the script that was trying to test the modules by running the scripts, since the packages got uninstalled and reinstalled. We can add these lines back but i felt like these scripts were more useful if the modules were not uninstalled at the end of it, barring also the time we save when we run [run_all_install_and_test.sh](https://github.com/entropicalabs/openqaoa/pull/236/files#diff-f9dffec80028b1c574b13e49e3365d8b489a64c8dca70d27d14147efc1ee599b)